### DISCUSSION TITLE
玉珏图极坐标系逆时针设置有问题（两个月过去了 - -）

### DISCUSSION THREAD

**Discussion by Break-2020** [antvis/G2#5111] [Q&A]
- [ ] I have searched the [issues](https://github.com/antvis/g2/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://g2-v4.antv.vision/zh/examples/bar/basic#radial-bar](https://g2-v4.antv.vision/zh/examples/bar/basic#radial-bar)

### Steps to reproduce
这个demo里，不管给极坐标系设置reflect('x')还是scale(1,-1)，图形都会出问题。
这个问题去年就有人提过，但是好像没有改
![image](https://user-images.githubusercontent.com/73464509/109443291-25456800-7a75-11eb-8151-10f101212f4f.png)

| Environment | Info |
|---|---|
| g2 | 4.1.12 |
| System | win10 |
| Browser | chrome |

**Comment by alphabetabc** [comment]
好像还在修复中[#3087](https://github.com/antvis/G2/pull/3087)

**Comment by Break-2020** [comment]
> 好像还在修复中[#3087](https://github.com/antvis/G2/pull/3087)

这个改动大概什么时候能用上呀

**Comment by yangyang996** [comment]
现在21年9月底了，还是没修复啊

**Comment by liu-minjie** [comment]
改好了吗  4.1.34里还是有问题

**Comment by yangyang996** [comment]
没改，放弃了，官方，我们关注了，没更新




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年5月9日(星期一) 下午5:30
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [antvis/G2] 玉珏图极坐标系逆时针设置有问题（两个月过去了 - -） (#3305)





 
改好了吗  4.1.34里还是有问题

**Comment by feijuncheng** [comment]
我已经打算换echarts了，ie上实在是玩不了，问题解决也很慢

**Comment by Break-2020** [comment]
> 我已经打算换echarts了，ie上实在是玩不了，问题解决也很慢

我们也准备换了

**Comment by hustcc** [comment]
G2 4.x 中这个问题因为底层设计原因解决不掉，在 G2 5.0 中已经修复这个 bug。见官网 [DEMO](https://g2.antv.antgroup.com/examples/general/radial/#apple-activity)。
